web/packages/shared/deepLinks.ts

```diff
@@ -67,7 +67,7 @@ export type ConnectMyComputerDeepURL = BaseDeepURL & {
 
 export type AuthenticateWebDeviceDeepURL = BaseDeepURL & {
   pathname: '/authenticate_web_device';
-  searchParams: { id: string; token: string };
+  searchParams: { id: string; token: string; redirect?: string };
```

Comment:
```suggestion
  searchParams: { id: string; token: string; redirect_uri?: string };
```

Thoughts on renaming this param? I'd rather have it consistent instead of calling it `redirect_uri` in the Web UI but `redirect` in the deep link.